assets/js/common/AdvisoryIcon/AdvisoryIcon.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { startCase } from 'lodash';
+import {
+  EOS_SHIELD_OUTLINED,
+  EOS_CRITICAL_BUG_OUTLINED,
+  EOS_QUESTION_MARK_FILLED,
+  EOS_ADD_BOX_OUTLINED,
+} from 'eos-icons-react';
+import classNames from 'classnames';
+
+import Tooltip from '@common/Tooltip';
+import { computedIconCssClass } from '@lib/icon';
+
+function MakeIcon({ type, centered = false, hoverOpacity = true, size = 'l' }) {
```

Comment:
Please rename this to just `Icon`, `MakeIcon` is very imperative and clumsy :smile: